src/styles/watch.scss

```diff
@@ -0,0 +1,63 @@
+
+.watch {
+  position: relative;
+  margin: 100px auto;
+  width: 600px;
+  height: 600px;
+
+  border: 10px solid cadetblue;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+
+  &__center {
+    position: absolute;
+    bottom: 47%;
+
+    width: 30px;
+    height: 30px;
+
+    background-color: brown;
+    border-radius: 50%;
+    z-index: 2;
+  }
+
+  &__minute {
```

Comment:
minute and second elements have the same styles. You can create one class watch__hand and modifiers watch__hand--second (to change some specific styles)

`
  <div class="watch__hand watch__hand--second"></div>
   <div class="watch__hand watch__hand--minute"></div>
`